content/en/docs/next/quickstart.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Quickstart
+weight: 900
+description: Get etcd up and running in less than 5 minutes!
+---
+
+## Step 1 - Install etcd
+
+Set the environment variables to help with the commands that follow.
+
+```
+ETCD_VER={{< param git_version_tag >}}
+
+GOOGLE_URL=https://storage.googleapis.com/etcd
+GITHUB_URL=https://github.com/etcd-io/etcd/releases/download
+```
+
+Download and install etcd:
+
+On Linux
+```
+$ DOWNLOAD_URL=${GITHUB_URL}
+
+$ rm -f /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+$ rm -rf /tmp/test-etcd && mkdir -p /tmp/test-etcd
+
+$ curl -L ${DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100   644  100   644    0     0   1396      0 --:--:-- --:--:-- --:--:--  1396
+100 12.5M  100 12.5M    0     0  9767k      0  0:00:01  0:00:01 --:--:-- 17.7M
+
+$ tar xzvf /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz -C /tmp/test-etcd --strip-components=1
+etcd-v3.5.0-alpha.0-linux-amd64/etcdctl
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/apispec/
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/apispec/swagger/
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/apispec/swagger/rpc.swagger.json
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/apispec/swagger/v3lock.swagger.json
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/apispec/swagger/v3election.swagger.json
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/README.md
+etcd-v3.5.0-alpha.0-linux-amd64/READMEv2-etcdctl.md
+etcd-v3.5.0-alpha.0-linux-amd64/README.md
+etcd-v3.5.0-alpha.0-linux-amd64/README-etcdctl.md
+etcd-v3.5.0-alpha.0-linux-amd64/etcd
+```
+
+On macOS
```

Comment:
I need to get the command output for the macOS section here. I don't expect it to be much different than above, but it'd be good to have.

Reply:
Or would it be clearer in this whole section to remove the command output? (since it isn't etcd output)

Reply:
Update: prompts in if showing output, prompts out if not.